### ISSUE TITLE
Note: ffmpeg problem, conda, Mac

### ISSUE THREAD

**Issue by keunwoochoi** [sigsep/sigsep-mus-db#14]
Just in case someone's having problem with ffmpeg with these kinds of error messages:
```python
>>> tracks[0].targets['vocals'].audio
dyld: Library not loaded: @rpath/libx264.138.dylib
  Referenced from: /Users/keunwooc/miniconda3/bin/ffprobe
  Reason: image not found
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/keunwooc/miniconda3/lib/python3.6/site-packages/musdb/audio_classes.py", line 113, in audio
    audio = source.audio
  File "/Users/keunwooc/miniconda3/lib/python3.6/site-packages/musdb/audio_classes.py", line 47, in audio
    filename=self.path, stem_id=self.stem_id
  File "/Users/keunwooc/miniconda3/lib/python3.6/site-packages/stempeg/read.py", line 91, in read_stems
    FFinfo = FFMPEGInfo(filename)
  File "/Users/keunwooc/miniconda3/lib/python3.6/site-packages/stempeg/read.py", line 19, in __init__
    self.json_info = read_info(self.filename)
  File "/Users/keunwooc/miniconda3/lib/python3.6/site-packages/stempeg/read.py", line 55, in read_info
    out = sp.check_output(cmd)
  File "/Users/keunwooc/miniconda3/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/Users/keunwooc/miniconda3/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['ffprobe', '../../Data/musdb18/train/A Classic Education - NightOwl.stem.mp4', '-v', 'error', '-print_format', 'json', '-show_format', '-show_streams']' died with <Signals.SIGABRT: 6>.
```
https://github.com/flatironinstitute/CaImAn/issues/317#issuecomment-405294541 help me by, I think, installing `libx264.138.dylib` with command below. 
```
$ conda install -c conda-forge x264=20131218
```

It's not really an issue but probably best to be written here.

**Comment by faroit** [comment]
This is weird. x264 should not be needed for decoding as this is an AAC stream. Which ffmpeg version have you installed? The Travis builds of stempeg use conda as well and are tested against  

```
- FFMPEG_VERSION=3.4

- FFMPEG_VERSION=2.8.6

- FFMPEG_VERSION=3.2.4

```

**Comment by keunwoochoi** [comment]
Interesting. My FFMPEG is `2.8.6` with miniconda python 3.6 and stempeg `0.1.3` - in case it matters.

**Comment by faroit** [comment]
i will try to reproduce this weekend

**Comment by faroit** [comment]
Actually. a fresh installation of ffmpeg by miniconda now points to ffmpeg 4.0.2. Also it indeed comes with x264 as a dependency:

<img width="765" alt="screen shot 2018-09-01 at 11 49 58" src="https://user-images.githubusercontent.com/72940/44944829-fd121b00-addd-11e8-85b4-555c2fa39dbf.png">

I tried it here on OS X... works. Also I've updated the travis test matrix. [All combinations seem to pass](https://travis-ci.org/faroit/stempeg/builds/423328765). Maybe try again with a fresh environment.
